src/main/java/io/stargate/sgv2/jsonapi/api/security/ErrorChallengeSender.java

```diff
@@ -28,18 +34,38 @@ public class ErrorChallengeSender implements ChallengeSender {
   /** Result is always constant */
   private final CommandResult commandResult;
 
+  /** The {@link MeterRegistry} to report to. */
+  private final MeterRegistry meterRegistry;
+
+  /** The tag for error being true, created only once. */
+  private final Tag errorTag = Tag.of("error", "true");
```

Comment:
These can probably be `static`? (unlike injected things like meterRegistry)

Reply:
Will add it. The class is marked application scope so only one instance is maintained.